lib/const.rb

```diff
@@ -57,6 +57,27 @@
   HOME = CREW_PREFIX + Dir.home
 end
 
+CREW_IN_CONTAINER = File.exist?('/.dockerenv') || !ENV['CREW_IN_CONTAINER'].to_s.empty?
+
+CREW_IS_AMD = `grep -c 'AuthenticAMD' /proc/cpuinfo`.to_i.positive?
+CREW_IS_INTEL = `grep -c 'GenuineIntel' /proc/cpuinfo`.to_i.positive?
```

Comment:
Combine to `CREW_CPU_VENDOR`?
```ruby
CREW_CPU_VENDOR = CPUINFO[:vendor_id] || 'unknown'
```

Reply:
Does CPUINFO need anything else to work?

Reply:
```
chronos@cheekon-x86_64 /usr/local/lib/crew/packages $ irb
irb(main):001:0> CPUINFO[:vendor_id]
(irb):1:in `<main>': uninitialized constant CPUINFO (NameError)
        from /usr/local/lib64/ruby/gems/3.1.0/gems/irb-1.4.1/exe/irb:11:in `<top (required)>'
        from /usr/local/bin/irb:25:in `load'
        from /usr/local/bin/irb:25:in `<main>'
irb(main):002:0>
```
Looks like we need some other gem? (The cpuinfo gem appears to be abandoned...)

Reply:
`const.rb`, line 9:
https://github.com/chromebrew/chromebrew/blob/7d3e2dc146798899f96d59a7e4123834d03a47be/lib/const.rb#L9

Reply:
(added in #7545)

Reply:
It wasn't working for me...

When I add `CREW_CPU_VENDOR = CPUINFO[:vendor_id] || 'unknown'`
```
chronos@cheekon-x86_64 /usr/local/lib/crew/packages $ crew const CREW_CPU_VENDOR
CREW_CPU_VENDOR=unknown
chronos@cheekon-x86_64 /usr/local/lib/crew/packages $ crew const CPUINFO
CPUINFO={"processor"=>"0", "vendor_id"=>"GenuineIntel", "cpu family"=>"6", "model"=>"94", "model name"=>"Intel(R) Core(TM) i3-6300T CPU @ 3.30GHz", "stepping"=>"3", "microcode"=>"0xf0", "cpu mhz"=>"3299.667", "cache size"=>"4096 KB", "physical id"=>"0", "siblings"=>"4", "core id"=>"0", "cpu cores"=>"2", "apicid"=>"0", "initial apicid"=>"0", "fpu"=>"yes", "fpu_exception"=>"yes", "cpuid level"=>"22", "wp"=>"yes", "flags"=>"fpu vme de pse tsc msr pae mce cx8 apic sep mtrr pge mca cmov pat pse36 clflush dts acpi mmx fxsr sse sse2 ss ht tm pbe syscall nx pdpe1gb rdtscp lm constant_tsc art arch_perfmon pebs bts rep_good nopl xtopology nonstop_tsc cpuid aperfmperf pni pclmulqdq dtes64 monitor ds_cpl vmx est tm2 ssse3 sdbg fma cx16 xtpr pdcm pcid sse4_1 sse4_2 x2apic movbe popcnt tsc_deadline_timer aes xsave avx f16c rdrand lahf_lm abm 3dnowprefetch cpuid_fault invpcid_single ssbd ibrs ibpb stibp tpr_shadow vnmi flexpriority ept vpid ept_ad fsgsbase tsc_adjust bmi1 avx2 smep bmi2 erms invpcid mpx rdseed adx smap clflushopt intel_pt xsaveopt xsavec xgetbv1 xsaves dtherm ida arat pln pts hwp hwp_notify hwp_act_window hwp_epp md_clear flush_l1d arch_capabilities", "vmx flags"=>"vnmi preemption_timer invvpid ept_x_only ept_ad ept_1gb flexpriority tsc_offset vtpr mtf vapic ept vpid unrestricted_guest ple pml", "bugs"=>"cpu_meltdown spectre_v1 spectre_v2 spec_store_bypass l1tf mds swapgs itlb_multihit srbds mmio_stale_data retbleed", "bogomips"=>"6599.98", "clflush size"=>"64", "cache_alignment"=>"64", "address sizes"=>"39 bits physical, 48 bits virtual"}
```

Reply:
```
chronos@cheekon-x86_64 /usr/local/lib/crew/packages $ irb
irb(main):001:0* CPUINFO = File.read('/proc/cpuinfo') \
irb(main):002:0*               .partition("\n\n")[0] \
irb(main):003:0*               .scan(/^(.+?)\t*: (.+)$/).to_h \
irb(main):004:0>               .transform_keys(&:downcase)
=> 
{"processor"=>"0",
...
irb(main):005:0> CREW_CPU_VENDOR = CPUINFO[:vendor_id] || 'unknown'
=> "unknown"
irb(main):006:0>
```

Reply:
I figured it out. I had to use `CPUINFO["vendor_id"]`, but this only works on x86...